presto-cassandra/src/test/java/io/prestosql/plugin/cassandra/TestCassandraClientConfig.java

```diff
@@ -100,9 +100,9 @@ public void testExplicitPropertyMappings()
                 .put("cassandra.speculative-execution.delay", "101s")
                 .put("cassandra.protocol-version", "V2")
                 .put("cassandra.tls.enabled", "true")
-                .put("cassandra.tls.keystore-path", "/tmp/keystore")
+                .put("cassandra.tls.keystore-path", "/dev/null")
```

Comment:
MInor thing: I am not a big fan of using same value for different keys as it may mask problem, though I do not have a good solution to that problem.

Reply:
I'm not either. That's why in the initial attempt I've created dummy files in src/test/resources - separate for each of the casses